cli/cli.go

```diff
@@ -62,13 +62,16 @@ var RootCmd = &cobra.Command{
 		go func(ch chan struct{}) {
 			for {
 				mu.Lock()
-				<-ch
+				_, isOpen := <-ch
+				if !isOpen {
+					mu.Unlock()
+					return
+				}
 				render(m)
 				mu.Unlock()
 			}
 		}(ch)
 		m.Run(ch, COUNT)
-		close(ch)
```

Comment:
Unless I'm missing something, I think this has no effect, as `Run` calls `discover` which closes the channel as the last thing it does.